src/sidebar/util/oauth-client.js

```diff
@@ -248,10 +247,10 @@ export default class OAuthClient {
     // key=value format.
     const authWindowSettings = queryString
       .stringify({
-        left,
-        top,
-        width,
-        height,
+        left: left.toString(),
```

Comment:
```suggestion
        left: `${left}`,
```

Reply:
I gently prefer `toString()` as it's more explicit.